assets/src/hooks/useKeyDown.tsx

```diff
@@ -11,7 +11,7 @@ export function useKeyDown(
 
     const ctrlMatch = ctrlRequired === undefined || ctrlKey === ctrlRequired;
     const metaMatch = metaRequired === undefined || metaKey === metaRequired;
-    if (ctrlMatch && metaMatch && keyCodes.includes(code)) {
+    if ((ctrlMatch || metaMatch) && keyCodes.includes(code)) {
```

Comment:
I don't think this makes sense to do. It shouldn't be an OR and I think that the ctrlKey should map to Command or something on a Mac  already

Reply:
Done.